web-app/js/portal/prototypes/Ext.js

```diff
@@ -34,5 +34,16 @@ Ext.tree.TreeNode.prototype.clone = function(recursive) {
     }
     
     return clone;
-}
+};
 
+Ext.tree.TreeNode.prototype.getHierarchy = function(attribute) {
```

Comment:
@jonescc I'm still looking at the code, however I wanted to ask why is this overridden at the Ext level. Wouldn't it be better to inherit from that class and then override?

This means that if TreeNode is used anywhere else - you get this non-standard behaviour as well.